### ISSUE TITLE
Google Sheets via docker

### ISSUE THREAD

**Issue by ScottAMains** [cpvalente/ontime#786]
Hi folks. 
Desktop app is great. Integrations work well. 
Having issues with replicating it in the docker environment. 

Creating a oAuth token (desktop client) downloading the JSON and uploading it to the docker install. Going through the credential validation I get to the point where Keep getting a redirect to localhost (my machine and not the docker machine) and a port number within the URL that is not exposed. 

Is there something I'm missing for the docker install for google sheets integration?

**Comment by cpvalente** [comment]
Hi @ScottAMains , thank you for reaching out

We recently realised that the Google Sheet flow does not work in docker.
This is because the host that authenticates the app is different from the docker container, exactly what you found. 

In v3 we are actively working on resolving this by changing how we authenticate against Google.
From the initial results we had, it seems promising and I believe the general flow is majorly improved.

In short, you have done everything correctly, and the issue is an oversight on our part.
We have a solution (#782) that is being merged to v3 in the next couple of days. Unfortunately, we do not plan on providing a fix for v2

**Comment by ScottAMains** [comment]
Thanks @cpvalente appreciate you reaching back. Looking forward to the next release. 
Keep up the great work.

**Comment by je71175** [comment]
Thank you. I had the same issue

**Comment by cpvalente** [comment]
@je71175 , I believe this may be [fixed in the alpha version of v3](https://hub.docker.com/layers/getontime/ontime/v3.0.0-alpha.2/images/sha256-f8e2606e0cad5e608ee8751128b582feb72cf18fa4a4be63476ba61d0aacb50f?context=explore). Would you be available to help test?

**Comment by cpvalente** [comment]
Hi @ScottAMains , would be interested in [testing out v3](https://github.com/cpvalente/ontime/releases/tag/v3.0.0-alpha.2) and giving us some feedback on how this works in your setup?

**Comment by cpvalente** [comment]
Fixed in v3
